security-analysis/security-analysis-api/src/main/java/com/powsybl/security/PostContingencyResult.java

```diff
@@ -22,9 +26,19 @@
 
     private final LimitViolationsResult limitViolationsResult;
 
+    Map<String, BranchResult> branchResults;
```

Comment:
We must add a test for JSON serialization / deserialization of the new results, too.
With the current code, I don't think the new result will be written --> to be tested :) 

Note that the result would be better serialized as a list, not a map, because otherwise the ID information will be repeated (once as a key, once as a field of the value). The `Map` is only useful to provide access to the results by their equipment ID.